src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
Correct me if I'm wrong, but I think that as it is now, when requiring it with Require.js, OpenSeadragon will end up leaked into the global scope, right? Probably better to change all the sources and do this properly? Which is just an opinion: I don't really know how complex that would be.